### ISSUE TITLE
Chunks still protected, but no longer in a claim.

### ISSUE THREAD

**Issue by harveydobson** [thexaero/open-parties-and-claims#193]
Hi all, we've been playing on a server with Valhelsia 5 (using open-parties-and-claims-forge-1.19.2-0.16.5.jar) for the last few days, and it's been great! Though I did run into a bit of an issue yesterday. The fist day I set-up the server, I put a land claim active around my base and had invited some friends to my party - it was all running smoothly but yesterday members of my party were unable to interact with it...  I tried everything, inviting them, uninviting them and eventually removing the claim all together, still whether they are in my party or not, they cannot interact with anything in my former claim... and further to that, they don't even get the error message that openpac normally throws when it is protecting a claim
It's almost as if it has modified those chunks in some way and forgotten about it. It possibly happened after a sever crash.. but I'm not 100% sure
Any idea how I would go about removing the protection on these chunks?  I've also tried reapplying and removing the claim information, with no success

So in summary:

- I cannot interact with anything on the claim without being an OP
- My party members cannot interact with anything on the claim without being an OP
- Non party members cannot interact with anything on the claim without being an OP

I have tried:

- Uninviting and reinviting party members
- Destroying the party and starting a new one
- Removing the claim
- Removing the claim and then re-adding it
- Setting the claim to a server claim and then re adding it

Any ideas?

**Comment by thexaero** [comment]
I don't think it's this mod protecting it. Is it around the spawn area or something? It could be vanilla spawn protection.

**Comment by ryleu** [comment]
Yeah, this seems like vanilla spawn protection. Go find the spawn protection radius in `server.properties` and make it 0.

**Comment by harveydobson** [comment]
@thexaero @ryleu Hi both, thanks so much for the reply, hadn't seen until just now. I had reached the same conclusion but didn't know I could change it in the server properties so I'll give that a go. 

It's odd as the spawn point didn't affect anything for the first few days. But I also realised it was only a subsection of my claim that had this issue.

Thanks again for your help, I'm confident this will be the issue.
